test/source/tests/setup.ts

```diff
@@ -447,6 +447,14 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
       await gmailPage.notPresent(['@webmail-notification-setup']);
     }));
 
+    ava.default('mail.google.com/chat', testWithBrowser(undefined, async (t, browser) => {
+      const acct = 'ci.tests.gmail@flowcrypt.dev';
+      const settingsPage = await BrowserRecipe.openSettingsLoginButCloseOauthWindowBeforeGrantingPermission(t, browser, acct);
+      await settingsPage.close();
+      const googleChatPage = await openMockGmailPage(t, browser, acct, false, TestUrls.mockGoogleChat());
+      await googleChatPage.notPresent(BrowserRecipe.oldAndNewComposeButtonSelectors); // compose button should not be injected
```

Comment:
I'm not sure what we're testing with the mock here,
especially `div.z0`. Why would it appear if we don't have such elements in our mock chat page?

Reply:
You are right. This. test is indeed not necessary for mock.
Just removed it.